main/admin/settings.lib.php

```diff
@@ -117,6 +117,97 @@ function handleExtensions()
     echo '<a class="btn btn-success" href="configure_extensions.php?display=ppt2lp" role="button">'.get_lang('Ppt2lp').'</a>';
 }
 
+/**
+ * Show form for plugin and validates inputs. Calls uploadPlugin() if everything OK
+ * @return string|void The HTML form, or displays a message and returns nothing on error
+ * @throws Exception
+ */
+function handlePluginUpload()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The variable $user_id is not named in camelCase.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5078559)